admin/assets/javascripts/admin/components/admin-edit-leaderboard.gjs

```diff
@@ -159,6 +160,13 @@ export default class AdminEditLeaderboard extends Component {
         </field.Custom>
       </form.Field>
 
+      <form.Field
+        @name="period_filter_disabled"
+        @title={{i18n "gamification.leaderboard.period_filter_disabled"}}
+        as |field|
+      >
+        <field.Checkbox @value{{field.value}} />
+      </form.Field>
```

Comment:
Is there a clean way to achieve an inline label with the checkbox without breaking out of form-kit? 

<img width="322" alt="Screenshot 2025-01-02 at 9 12 02 PM" src="https://github.com/user-attachments/assets/c9dcdcd9-e8f7-4fe3-a3cd-c1745d4a8f71" />

Setting the `@showTitle` argument on the field generates the following, but it lacks the optional tag and does not match the styling of a standard form label. Could I style it myself? Yes! But I wanted to confirm whether this isn’t possible out of the box first.

<img width="400" alt="Screenshot 2025-01-02 at 9 14 08 PM" src="https://github.com/user-attachments/assets/7e927bd9-ea75-4d2c-9f07-3e0cd765059f" />